torch/csrc/jit/python/python_sugared_value.cpp

```diff
@@ -886,6 +886,11 @@ std::shared_ptr<SugaredValue> toSugaredValue(
       obj.ptr() ==
       py::module::import("torch.distributed.rpc").attr("rpc_async").ptr()) {
     return SpecialFormValue::create(prim::rpc_async);
+  } else if (
+      // RPC module is only avaialble  when build flag "USE_DISTRIBUTED" is on.
+      obj.ptr() ==
+      py::module::import("torch.distributed.rpc").attr("rpc_sync").ptr()) {
```

Comment:
is the indent here intentional?

Reply:
it's formatted by clang-format (and stay aligned with the above rpc_async op)